code/modules/ai/ai_holder.dm

```diff
@@ -318,9 +318,11 @@
 		if(STANCE_MOVE)
 			if(hostile && find_target()) // This will switch its stance.
 				ai_log("handle_stance_strategical() : STANCE_MOVE, found target and was inturrupted.", AI_LOG_TRACE)
+				return
 		if(STANCE_FOLLOW)
 			if(hostile && find_target()) // This will switch its stance.
 				ai_log("handle_stance_strategical() : STANCE_FOLLOW, found target and was inturrupted.", AI_LOG_TRACE)
+				return
```

Comment:
It's to prevent compile warnings about if() statements having no effect because of the nullified ai_log define. I can  handle it another way if you want.